WordPress/Classes/Services/PostService+Likes.swift

```diff
@@ -0,0 +1,68 @@
+extension PostService {
+
+    /**
+     Fetches a list of users that liked the post with the given ID.
+     
+     @param postID  The ID of the post to fetch likes for
+     @param siteID  The ID of the site that contains the post
+     @param success A success block
+     @param failure A failure block
+     */
+    func getLikesFor(postID: NSNumber,
+                     siteID: NSNumber,
+                     success: @escaping (([LikeUser]?) -> Void),
+                     failure: ((Error?) -> Void)? = nil) {
```

Comment:
Curious why a default nil for the failure block.  Seems like we'd always want to handle failure?  Maybe this is related to future plans?
Also, should this be flagged `@escaping` as well?

Reply:
> Curious why a default nil for the failure block. Seems like we'd always want to handle failure? Maybe this is related to future plans?

Yep. I haven't handled failures yet, so I'm not sure if this should be required or not. 

> should this be flagged @escaping as well?

Ah, I did try, but Swifty gets all complainy.

<img width="1405" alt="Screen Shot 2021-04-26 at 4 58 04 PM" src="https://user-images.githubusercontent.com/1816888/116161353-dd2b7600-a6b0-11eb-8c86-ba094e6da30c.png">

Reply:
Nevermind. I went ahead and changed it. 😄